stackdriver/metric_client.go

```diff
@@ -18,7 +18,9 @@ import (
 	"context"
 
 	monitoring "cloud.google.com/go/monitoring/apiv3"
+	"github.com/google/ts-bridge/version"
```

Comment:
nit: please separate internal imports and external imports, see: https://github.com/google/ts-bridge/blob/c0e8016838f2da717e4c672cf1bd8220fb9ff73f/app/main.go#L28